app/views/schools/core_programme/materials/show.html.erb

```diff
@@ -0,0 +1,20 @@
+<% content_for :before_content, govuk_back_link( text: "Back", href: schools_cohort_path(@cohort.start_year)) %>
+
+<span class="govuk-caption-l"><%= @cohort.start_year %> cohort</span>
+<h1 class="govuk-heading-l">Choose what training materials you want to use for this cohort</h1>
+
+<p><strong>Current training materials:</strong> <%= @school_cohort.core_induction_programme.name %></p>
+
+<h2 class="govuk-heading-m">Your options if you want to change</h2>
+<p>Choose the following materials</p>
+<ul class='govuk-list govuk-list--bullet'>
+  <% CoreInductionProgramme.all.without(@school_cohort.core_induction_programme).each do |cip| %>
+    <li><%= govuk_link_to cip.name, '#' %></li>
```

Comment:
Link to nowhere?

Reply:
Yes, those really have nowhere to go. This page was created out of the wireframe without content file, I might need to dig into this a bit more. Show stopper for now? Convert to text instead?

Reply:
You can also add a todo if the page it links to has not been implemented